core/tests/integration/inputs/contracts/contracts.ncl

```diff
@@ -1,5 +1,5 @@
 # test.type = 'pass'
-let {check, Assert, ..} = import "../lib/assert.ncl" in
+
```

Comment:
As we use `Assert` a lot, I wonder if we shouldn't still alias it, with `let Assert = std.contract.Assert` instead of repeating the `std.contract` part everywhere. We usually use `check` only once, so `check` is fine as it is.

Reply:
Sure, we can do that